pkgs/development/tools/drip/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, gcc, jdk, which }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "drip";
+  version = "0.2.4";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "flatland";
```

Comment:
Not that I know anything about this particular package but is there a reason to use a fork that appears to be behind the parent repo?